app/src/main/java/com/orgzly/android/sync/AutoSync.kt

```diff
@@ -43,10 +48,11 @@ class AutoSync @Inject constructor(val context: Application, val dataRepository:
     enum class Type {
         NOTE_CREATED,
         DATA_MODIFIED,
-        APP_RESUMED
+        APP_RESUMED,
+        APP_SUSPEND,
```

Comment:
Longer, but I think we should keep the same tense as the other actions so it's easier to read the code
```suggestion
        APP_SUSPENDED,
```